src/event.js

```diff
@@ -30,7 +30,14 @@ function returnFalse() {
 }
 
 // Support: IE <=9 only
-// See #13393 for more info
+// https://bugs.jquery.com/ticket/13393
+function expectSync( elem, type ) {
+
+	// Expect focus to be synchronous when the element is already active,
+	// and blur to be synchronous when the element is not already active
+	// (even in IE)
+	return ( elem === safeActiveElement() ) === ( type === "focus" );
```

Comment:
You could perhaps add a support comment for IE <=11 here as other browsers fire those events synchronously so this helper wouldn't be needed?

It seems to me the `Support: IE <=9 only` part should be kept directly above `safeActiveElement` and `expectSync` might get a `Support: IE <=11 only` comment (or `Support: IE <=11+`, whatever we decide the convention to be).

What do you think?

Reply:
Good points; done.